doc/cci_compatibility_guide.md

```diff
@@ -0,0 +1,730 @@
+    </ShellConfig>
+    ```
+    for node3
+    ```bash
+    $ cat ./CQT/configuration/Function_Db/shell_config.xml
+    <ShellConfig>
+    <ip>localhost</ip>
+    <port>8516</port>
+    </ShellConfig>
+    ```
+ 
+* Touch start_test.sh       
+    ```bash
+    #!/bin/sh
+    cd $HOME/CTP/common/script
+    sh upgrade.sh
+    cd $HOME
+    stop_consumer.sh 
+    rm -f nohup.out
+    nohup start_consumer.sh -q QUEUE_CUBRID_QA_COMPAT_CCI_SHELL_DRIVER_64,QUEUE_CUBRID_QA_COMPAT_CCI_SHELL_SERVER_64 -exec run_compat_cci,run_compat_cci -s china &
+    ```
+* Check out test cases        
+We need to prepare test case of both high version and low version.      
+  * For high version's server(>=10.0)    
+    ```bash
+    cd ~
+    git clone https://github.com/CUBRID/cubrid-testcases-private.git 
+    ```
+  * For low version's server(<10.0)     
+    ```bash
+    cd ~
+    mkdir dailyqa
+    cd dailyqa
+    svn co https://oss.navercorp.com/CUBRID/cubridqa/branches/RB-8.4.1
+    svn co https://oss.navercorp.com/CUBRID/cubridqa/branches/RB-8.4.3
+    svn co https://oss.navercorp.com/CUBRID/cubridqa/branches/RB-8.4.4
+    svn co https://oss.navercorp.com/CUBRID/cubridqa/branches/RB-9.2.0
+    svn co https://oss.navercorp.com/CUBRID/cubridqa/branches/RB-9.3.0
+    ```
+* Configure .bash_profile    
+    ```
+    export QA_REPOSITORY=$HOME/CQT
+    export PROPERTIES_PATH=$QA_REPOSITORY/qatool_bin/qamanager
+    export init_path=$QA_REPOSITORY/lib/shell/common
+    export SHELL_CONFIG_PATH=$QA_REPOSITORY/lib/shell/common
+
+
+    export DEFAULT_BRANCH_NAME=develop
+    export CTP_HOME=$HOME/CTP
+    export CTP_BRANCH_NAME=develop
+    export CTP_SKIP_UPDATE=0
+
+    ulimit -c unlimited
+    export LC_ALL=en_US
+
+    . $HOME/.cubrid.sh
+    export CLASSPATH=.:$JAVA_HOME/lib/:$QA_REPOSITORY/qatool_bin/console/for_jdk6/cubridqa-console.jar:$CUBRID/jdbc/cubrid_jdbc.jar:$QA_REPOSITORY/lib/shell/common/commonforjdbc.jar
+    export PATH=$CTP_HOME/bin:$CTP_HOME/common/script:$JAVA_HOME/bin:/usr/local/bin:/bin:/usr/bin:$PATH
+
+
+
+
+    #-------------------------------------------------------------------------------
+    # set CUBRID environment variables
+    #-------------------------------------------------------------------------------
+    . /home/ccompat3/.cubrid.sh
+    ```
+
+* Install necessary shell commands     
+  cci test cases use killall command, centos7 can not find this command
+   ```bash
+   yum install psmisc
+   ```    
+# 4. Regression Tests
+We perform cci compatibility test for each build.     
+## 4.1 Daily Regresion Test
+When a new build comes, the test will start. We just need to make sure that test environment has no problem and listener has been started. Sometimes, in order to investigate or correct a test, we need to send messages manually.
+### Start the listener
+```bash
+$ cd ~
+$ sh start_test.sh &
+$ tail -f nohup.out
+```
+### Send test message
+#### Configuration 
+There are configuration files for compatibility messages
```

Comment:
We need add  full configuration about CCI (also JDBC) compatiblity test messages.

* CTP/conf/job.conf

		job_compat.service=ON
		job_compat.crontab=0/5 * * * * ?
		job_compat.listenfile=CUBRID-{1}-linux.x86_64.sh
		job_compat.acceptversions=10.0.*.0~8999,10.1.*,10.2.*
		job_compat.package_bits=64
		job_compat.package_type=general

		...
		...

		job_compat.test.3.scenario=compat_cci_D64
		job_compat.test.3.queue=QUEUE_CUBRID_QA_COMPAT_CCI_SHELL_DRIVER_64
		job_compat.test.3.ext_config=compat.conf
		job_compat.test.3.ext_keys=cci_compatibility_for_{version}_D64

		job_compat.test.4.scenario=compat_cci_S64
		job_compat.test.4.queue=QUEUE_CUBRID_QA_COMPAT_CCI_SHELL_SERVER_64
		job_compat.test.4.ext_config=compat.conf
		job_compat.test.4.ext_keys=cci_compatibility_for_{version}_S64

		...
		...


* CTP/conf/compat.conf

		EXT_KEY_FOR_FIX_MAX=COMPAT_BUILD_ID
		EXT_KEY_FOR_FIX_MAX_FOLLOW=COMPAT_BUILD_URLS
		EXT_KEY_FOR_FIX_MAX_FOLLOW_KR=COMPAT_BUILD_URLS_KR
		EXT_KEY_FOR_TEST_CATAGORY=COMPAT_TEST_CATAGORY

		...
		...

		cci_compatibility_for_10.2.0_D64=cci_shell_10.1_S64.msg,cci_shell_10.0_S64.msg

		cci_compatibility_for_10.1.0_D64=cci_shell_10.2_S64.msg,cci_shell_10.0_S64.msg

		cci_compatibility_for_10.0.0_D64=cci_shell_10.2_S64.msg,cci_shell_10.1_S64.msg

		cci_compatibility_for_10.2.0_S64=cci_shell_10.1_D64.msg,cci_shell_10.0_D64.msg,cci_shell_9.3_D64.msg,cci_shell_9.2_D64.msg,cci_shell_8.4.4_D64.msg,cci_shell_8.4.3_D64.msg,cci_shell_8.4.1_D64.msg,

		cci_compatibility_for_10.1.0_S64=cci_shell_10.2_D64.msg,cci_shell_10.0_D64.msg,cci_shell_9.3_D64.msg,cci_shell_9.2_D64.msg,cci_shell_8.4.4_D64.msg,cci_shell_8.4.3_D64.msg,cci_shell_8.4.1_D64.msg,

		cci_compatibility_for_10.0.0_S64=cci_shell_10.2_D64.msg,cci_shell_10.1_D64.msg,cci_shell_9.3_D64.msg,cci_shell_9.2_D64.msg,cci_shell_8.4.4_D64.msg,cci_shell_8.4.3_D64.msg,cci_shell_8.4.1_D64.msg,


* Detail compat configuration

		~/CTP/conf/compat/cci_shell_8.4.3_D64.msg
		COMPAT_BUILD_ID=8.4.3.{max}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_10.0_S64.msg
		COMPAT_BUILD_ID=10.0.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SVN_BRANCH=unknown
		COMPAT_BUILD_SCENARIOS=cci
		COMPAT_BUILD_SCENARIO_BRANCH_GIT=release/10.0

		~/CTP/conf/compat/cci_shell_10.1_D64.msg
		COMPAT_BUILD_ID=10.1.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-Linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_10.2_D64.msg
		COMPAT_BUILD_ID=10.2.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-Linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_9.1_D64.msg
		COMPAT_BUILD_ID=9.1.0.0212
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_9.2_D64.msg
		COMPAT_BUILD_ID=9.2.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_10.1_S64.msg
		COMPAT_BUILD_ID=10.1.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-Linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SVN_BRANCH=unknown
		COMPAT_BUILD_SCENARIOS=cci
		COMPAT_BUILD_SCENARIO_BRANCH_GIT=release/10.1

		~/CTP/conf/compat/cci_shell_10.2_S64.msg
		COMPAT_BUILD_ID=10.2.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-Linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SVN_BRANCH=unknown
		COMPAT_BUILD_SCENARIOS=cci
		COMPAT_BUILD_SCENARIO_BRANCH_GIT=develop

		~/CTP/conf/compat/cci_shell_10.0_D64.msg
		COMPAT_BUILD_ID=10.0.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_9.0_D64.msg
		COMPAT_BUILD_ID=9.0.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_8.4.5_D64.msg
		COMPAT_BUILD_ID=8.4.5.{max}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_8.4.4_D64.msg
		COMPAT_BUILD_ID=8.4.4.{max}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_9.3_D64.msg
		COMPAT_BUILD_ID=9.3.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_9.4_D64.msg
		COMPAT_BUILD_ID=9.4.{dmax}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci

		~/CTP/conf/compat/cci_shell_8.4.1_D64.msg
		COMPAT_BUILD_ID=8.4.1.{max}
		COMPAT_BUILD_URLS=CUBRID-{BUILD_ID}-linux.x86_64.sh
		COMPAT_BUILD_BIT=64
		COMPAT_BUILD_TYPE=general
		COMPAT_BUILD_SCENARIOS=cci